accepted/2025/cli-acquisition-tool.md

```diff
@@ -0,0 +1,224 @@
+# .NET SDK Acquisition and Management Tool
+
+**Owner** [Chet Husk](https://github.com/baronfel) | [Daniel Plaisted](https://github.com/dsplaisted)
+
+Users of the .NET SDK today face a wide array of choices when getting our tooling, with multiple distribution channels that release at different cadences.
+While these options serve specific needs, several important user cohorts are not well served and have asked for a more unified approach to getting started with our tooling.
+
+
+This proposal introduces a standalone install and SDK management tool (tentatively named `dnup`) that provides a unified interface for acquiring and managing .NET tooling across all supported platforms, focusing on user-local installations that can be managed (largely) without elevation.
+
+This tool is especially critical for users that are
+
+* new users, especially those from the Node or Rust ecosystems
+* Linux users who typically use packages from their distro, as not all distro maintainers support feature bands beyond 1xx
```

Comment:
source build does not support > 1xx prior to 10 so this isn't possible at all today.  Starting in 10, distro maintainers will have the ability to support which ever feature bands they see a business need for.